### ISSUE TITLE
How should we handle new component requests from teams?

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#3489]

**Comment by gfellerph** [comment]
- Component request issue template
- What's the idea behind the request?
- Different reason why we could implement a component that should be included (accessibility, shareability). New components always need a discussion in the team
- Design System should act as an enabler
- Use Teams Channel to let the community participate on new components and ask for ideas/material, also use Open Design System Office Hours
- Stakeholders should know the way the Design System works (enabler vs. single source of truth)
- When request comes in, ask "what's currently missing in the System that you need to build it"
- How do we decide if the functionality should be implemented by the Design System?
- Different requests with different background thoughts
- Whenever a request comes in, we need to discuss about it
- Who's needed to make pattern decisions? Could be a small group that then presents the decision?
- It's pretty important that everybody knows what's going on
- We can use the Pattern Discussion to ask who's interested into discussing new components
- New components should always have at least a dev and a designer assigned, but can also have more than one
- Could sometimes be nice to organize a small coffee meeting with the request author or original dev, designer

**Actions**:
- Improve rules
- Create issue template specific to new components requests
